MapboxNavigation/RouteVoiceController.swift

```diff
@@ -60,6 +60,12 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     override public init() {
         super.init()
+        
+        if !Bundle.main.backgroundModes.audio {
+            print("Voice guidance may not work properly. " +
```

Comment:
The voice controller itself and the background mode is somewhat optional so I'd prefer to keep this subtle debug message.